### PR TITLE
chore: remove instances of CompatChrome

### PR DESCRIPTION
**Pull request by schalkneethling**
This removes instances of `CompatChrome` from the content. It also updates some sections to be closer to mdn/content.
Your feedback is appreciated.

Part of #5618

### CODE REVIEW

File: files/ja/web/api/audiobuffersourcenode/index.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: AudioBufferSourceNode
 slug: Web/API/AudioBufferSourceNode
+browser-compat: api.AudioBufferSourceNode
```
Comment (mfuji09) [inline files/ja/web/api/audiobuffersourcenode/index.md:4]: According to #7412, the `browser-compat` metadata should be deleted from translated contents, but why is this line added?

File: files/es/web/api/window/prompt/index.md [hunk 2]
```diff
@@ -42,29 +43,37 @@ El ejemplo anterior muestra el siguiente cuadro de diálogo (en Chrome en OS X):
 
 [![prompt() dialog in Chrome on OS X](https://mdn.mozillademos.org/files/11303/prompt.png)](https://mdn.mozillademos.org/files/11303/prompt.png)
 
-## Notas
+## Notes
```
Comment (Graywolf9) [inline files/es/web/api/window/prompt/index.md:46]:
```suggestion
## Notas
```

File: files/es/web/api/window/prompt/index.md [hunk 3]
```diff
@@ -42,29 +43,37 @@ El ejemplo anterior muestra el siguiente cuadro de diálogo (en Chrome en OS X):
...
 
-Un diálogo prompt contiene un cuadro de texto de una línea, un botón Cancel (Cancelar) un botón OK (Aceptar), y devuelve el texto (posiblemente vacío) que el usuario introdujo en el cuadro de texto.
+A prompt dialog contains a single-line textbox, a Cancel button, and an OK button, and
+returns the (possibly empty) text the user entered into that textbox.
```
Comment (Graywolf9) [inline files/es/web/api/window/prompt/index.md:49]:
```suggestion
Un diálogo prompt contiene un cuadro de texto de una línea, un botón Cancel (Cancelar) un botón OK (Aceptar), y devuelve el texto (posiblemente vacío) que el usuario introdujo en el cuadro de texto.
```

File: files/es/web/api/window/prompt/index.md [hunk 4]
```diff
@@ -42,29 +43,37 @@ El ejemplo anterior muestra el siguiente cuadro de diálogo (en Chrome en OS X):
...
 
-The following text is shared between this article, DOM:window\.confirm and DOM:window\.alertLos cuadros de diálogo son ventanas modales; previenen que el usuario acceda al resto de la interfaz del programa hasta que el cuadro de diálogo es cerrado. Por esta razón, no se debe abusar de cualquier función que crea un cuadro de diálogo (o ventana modal).
+Please note that result is a string. That means you should sometimes cast the value
+given by the user. For example, if their answer should be a Number, you should cast the
+value to Number.
```
Comment (Graywolf9) [inline files/es/web/api/window/prompt/index.md:53]:
```suggestion
Nótese que el resultado es una cadena de texto. Esto significa que a veces se deberá hacer una conversión al valor introducido por el usuario. Por ejemplo, si la respuesta debe ser un valor numérico, se debe hacer la conversión del valor a tipo Number.
```

File: files/es/web/api/window/print/index.md [hunk 5]
```diff
@@ -23,22 +24,28 @@ Abre el _Diálogo de Impresión_ para imprimir el documento actual.
...
 ## Sintaxis
 
-```js
-window.print()
+```js-nolint
+print()
 ```
 
-## Notas
+### Parameters
```
Comment (Graywolf9) [inline files/es/web/api/window/print/index.md:31]:
```suggestion
### Parámetros
```

File: files/es/web/api/window/print/index.md [hunk 6]
```diff
@@ -23,22 +24,28 @@ Abre el _Diálogo de Impresión_ para imprimir el documento actual.
...
 
-Empezando con Chrome {{CompatChrome(46.0)}} este método esta bloqueado dentro de un {{htmlelement("iframe")}} a menos que el atributo del contenedor tenga el valor `allow-modal`.
+None.
```
Comment (Graywolf9) [inline files/es/web/api/window/print/index.md:33]:
```suggestion
Ninguno.
```

File: files/es/web/api/window/print/index.md [hunk 7]
```diff
@@ -23,22 +24,28 @@ Abre el _Diálogo de Impresión_ para imprimir el documento actual.
...
 
-## Especificación
+### Return value
```
Comment (Graywolf9) [inline files/es/web/api/window/print/index.md:35]:
```suggestion
### Valor devuelto
```

File: files/es/web/api/window/print/index.md [hunk 8]
```diff
@@ -23,22 +24,28 @@ Abre el _Diálogo de Impresión_ para imprimir el documento actual.
...
 
-| Especificación                                                                       | Estatus                                      | Comentario          |
-| ------------------------------------------------------------------------------------ | -------------------------------------------- | ------------------- |
-| {{SpecName('HTML5 Web application', '#dom-print', 'print()')}} | {{Spec2('HTML5 Web application')}} | Initial definition. |
+None ({{jsxref("undefined")}}).
```
Comment (Graywolf9) [inline files/es/web/api/window/print/index.md:37]:
```suggestion
Ninguno ({{jsxref("undefined")}}).
```

File: files/es/web/api/window/print/index.md [hunk 9]
```diff
@@ -23,22 +24,28 @@ Abre el _Diálogo de Impresión_ para imprimir el documento actual.
...
 
-## Ver también
+## Specifications
```
Comment (Graywolf9) [inline files/es/web/api/window/print/index.md:39]:
```suggestion
## Especificaciones
```

File: files/es/web/api/window/print/index.md [hunk 10]
```diff
@@ -23,22 +24,28 @@ Abre el _Diálogo de Impresión_ para imprimir el documento actual.
...
 
-- [Impresión](/en/Printing "en/Printing")
-- {{ domxref("window.onbeforeprint") }}
-- {{ domxref("window.onafterprint") }}
+{{Specifications}}
+
+## Browser compatibility
```
Comment (Graywolf9) [inline files/es/web/api/window/print/index.md:43]:
```suggestion
## Compatibilidad con navegadores
```

File: files/es/web/api/window/print/index.md [hunk 11]
```diff
@@ -23,22 +24,28 @@ Abre el _Diálogo de Impresión_ para imprimir el documento actual.
...
+
+{{Compat}}
+
+## See also
```
Comment (Graywolf9) [inline files/es/web/api/window/print/index.md:47]:
```suggestion
## Véase también
```

File: files/es/web/api/window/prompt/index.md [hunk 12]
```diff
@@ -42,29 +43,37 @@ El ejemplo anterior muestra el siguiente cuadro de diálogo (en Chrome en OS X):
...
+Dialog boxes are modal windows; they
+prevent the user from accessing the rest of the program's interface until the dialog box
+is closed. For this reason, you should not overuse any function that creates a dialog
+box (or modal window).
 
-A partir de Chrome {{CompatChrome(46.0)}} este método está bloqueado para los elementos {{htmlelement("iframe")}}, , a menos que su atributo [sandbox](/es/docs/Web/HTML/Elemento/iframe#attr-sandbox) tenga el valor `allow-modal`.
+Alternatively {{HTMLElement("dialog")}} element can be used to take user inputs.
 
-En Safari, si el usuario presiona el botón Cancel, la función devuelve una cadena vacía. Por lo tanto, no se puede diferenciar si canceló o si mandó una cadena de texto vacía como valor del cuadro de texto.
+## Specifications
```
Comment (Graywolf9) [inline files/es/web/api/window/prompt/index.md:66]:
```suggestion
## Especificaciones
```

File: files/es/web/api/window/prompt/index.md [hunk 13]
```diff
@@ -42,29 +43,37 @@ El ejemplo anterior muestra el siguiente cuadro de diálogo (en Chrome en OS X):
...
 
-Esta función no tiene efecto en la versión Modern UI/Metro de Internet Explorer para Windows 8. No se muestra un diálogo al usuario, y siempre devuelve `undefined`. No está claro si esto es un bug o un comportamiento previsto. Las versiones de escritorio de IE sí implementan esta función
+{{Specifications}}
 
-## Especificación
+## Browser compatibility
```
Comment (Graywolf9) [inline files/es/web/api/window/prompt/index.md:70]:
```suggestion
## Compatibilidad con navegadores
```

File: files/es/web/api/window/prompt/index.md [hunk 14]
```diff
@@ -42,29 +43,37 @@ El ejemplo anterior muestra el siguiente cuadro de diálogo (en Chrome en OS X):
...
 
-| Especificación                                                                       | Estado                                       | Comentarios        |
-| ------------------------------------------------------------------------------------ | -------------------------------------------- | ------------------ |
-| {{SpecName('HTML5 Web application', '#dom-prompt', 'prompt()')}} | {{Spec2('HTML5 Web application')}} | Definición inicial |
+{{Compat}}
 
-## Véase también
+## See also
```
Comment (Graywolf9) [inline files/es/web/api/window/prompt/index.md:74]:
```suggestion
## Véase también
```

**Review by Graywolf9 — COMMENTED** [review]
Thank you so much. I let you some comments

**Review by josielrocha — APPROVED** [review]

**Comment by Graywolf9** [comment]
@schalkneethling hope you don't mind I applied the minnor changes from `es`,  I was trying to solve only the conflics for `es` but didn't let me do only `es` and I always have some problems solving old commits conflicts 😅

Also for the conflicts you just keep your changes over main

**Review by sashasushko — APPROVED** [review]
Thanks a lot 🤝 Approve from @mdn/yari-content-ru

**Comment by sashasushko** [comment]
@schalkneethling do you need help with resolving conflicts?

File: files/ja/web/api/audiobuffersourcenode/index.md (hunk 1 above)
Comment (schalkneethling) [reply files/ja/web/api/audiobuffersourcenode/index.md:4]: Ah, I might have seen this in the English content, not sure to be honest :)

File: files/es/web/api/window/print/index.md [hunk 15]
```diff
@@ -9,6 +9,7 @@ tags:
   - Referencia
   - Window
   - metodo
+browser-compat: api.Window.print
```
Comment (schalkneethling) [inline files/es/web/api/window/print/index.md:12]: @mfuji09 So these are not needed then?

Comment (yin1999) [reply files/es/web/api/window/print/index.md:12]: IMO, we can just remove them, the BCD and specification macros are still working.

See: https://github.com/mdn/yari/blob/5d8d88976164b6f653c3f2de688f3cd76f8537bb/kumascript/index.js#L56-L69

The parent document's metadata will be merged into the reflected translated document's metadata.

Comment (schalkneethling) [reply files/es/web/api/window/print/index.md:12]: Yup, I noticed that they _do_ work without these entries. I will clean it up.

Comment (schalkneethling) [inline files/es/web/api/window/print/index.md:12]:
```suggestion

```

File: files/es/web/api/window/prompt/index.md [hunk 16]
```diff
@@ -4,6 +4,7 @@ slug: Web/API/Window/prompt
 tags:
   - Referencia
   - metodo
+browser-compat: api.Window.prompt
```
Comment (schalkneethling) [inline files/es/web/api/window/prompt/index.md:7]:
```suggestion

```

File: files/ja/web/api/audiobuffersourcenode/index.md (hunk 1 above)
Comment (schalkneethling) [inline files/ja/web/api/audiobuffersourcenode/index.md:4]:
```suggestion
```

File: files/pt-br/web/api/window/print/index.md [hunk 17]
```diff
@@ -5,6 +5,7 @@ tags:
   - DOM
   - Window
   - print
+browser-compat: api.Window.print
```
Comment (schalkneethling) [inline files/pt-br/web/api/window/print/index.md:8]:
```suggestion
```

File: files/es/web/api/window/print/index.md (hunk 15 above)
Comment (schalkneethling) [reply files/es/web/api/window/print/index.md]: Removed.

**Review by sashasushko — APPROVED** [review]

**Review by Graywolf9 — APPROVED** [review]

**Comment by Graywolf9** [comment]
Is it only missing the approval of `ja` @mfuji09 ?

**Comment by sashasushko** [comment]
@schalkneethling do you need help with conflicts in `ru` locale?

**Comment by mfuji09** [comment]
I have updated the base of this PR.
The `ja` and `es` locales have been fixed by other PRs.
The `ru` locale has been converted to markdown, so the change has been invalidated.

**Review by yin1999 — COMMENTED** [review]
There is no changes on pt-BR docs since the last review. Let's merge this
